specs/create-todo.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test';
+import { TodoPage } from '../pages/TodoPage';
+
+test('Add To Do', async({page})=>{
+    const todoPage = new TodoPage(page);
+
+    await todoPage.goto();
+
+    await todoPage.addToDoItem('title 1', 'description 1');
+
+    const ToDoItemList = await page.evaluate(()=> {
+        return JSON.parse(localStorage.getItem('todolist') || '[]')
```

Comment:
Consider move the logic to the separate utility function / service class.
After you can re-use the function everywhere without code duplication.

Check this article: https://fullscale.io/blog/software-engineering-principles/
Especially `DRY` principle